microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -682,20 +679,67 @@ See also:
 * [Filter requests](../network/reference.md#filter-requests) in _Network features reference_.
 
 Status:
-* This checkbox is present in Microsoft Edge Canary 120.
-* This checkbox is not present in Microsoft Edge Stable 118.
+* This checkbox is present in Microsoft Edge Canary 121.
+* This checkbox is present in Microsoft Edge Stable 119.
+
+
+<!-- ====================================================================== -->
+## Enable breadcrumbs in the Performance Panel
+<!-- no (?) link -->
+
+_new as of Microsoft Edge 121_
+
+Adds a timing-breadcrumbs path in the **Application** tool, such as:
+
+Full range (5111.69ms) > 437.30ms > 46.52ms > 7.44ms
+
+![The timing breadcrumbs path](./index-images/timing-breadcrumbs-path.png)
```

Comment:
1 - @captainbrosset - can you create a final-quality screenshot png?  else will delete png.